src/Pdf.php

```diff
@@ -168,4 +177,43 @@ public function create()
 
         return true;
     }
+
+    /**
+     * @param  $fileName
+     * @return void
+     */
+    private function previewFileName($fileName = null)
+    {
+        if ($fileName && $fileName !== $this->fileName) {
```

Comment:
`$fileName !== $this->fileName` is not needed in my eyes. The method `previewFileName` should set the provided file name no matter what.